### DISCUSSION TITLE
Vierstra motif archetypes are now natively supported

### DISCUSSION THREAD

**Discussion by rcorces** [GreenleafLab/ArchR#1386] [Announcements]
Jeff Vierstra has produced "motif archetypes" which represent clustered motifs that have been essentially deduplicated based on similarity. In the [original v1 release](https://www.vierstra.org/resources/motif_clustering), he clustered the motifs and then annotated each individual motif with the name of the cluster (or "archetype") that it clustered with. Now, in the [beta v2 release](https://resources.altius.org/~jvierstra/projects/motif-clustering-v2.0beta/), he has produced PWM models for each of the archetype motif clusters which allows for searching of these archetypes explicitly.

ArchR now supports the use of the "Vierstra" motif databases in the `addMotifAnnotations()` function natively on the `release_1.0.2` branch. You can get these motifs using `motifSet="vierstra"` and `collection="archetype"` for the v2 archetype clustered pwms or `collection = "individual"` for the previous v1 individual pwms. 

You can see his Twitter post about this here - https://twitter.com/jeffvierstra/status/1514973966045048835?s=20&t=TfKlHoUT7M1kWpMXQlYFZA

You can find the code he used to do this here - https://github.com/jvierstra/motif-clustering

Code changes to `addMotifAnnotations()` here - https://github.com/GreenleafLab/ArchR/pull/1385

**Comment by eliascrapa** [comment]
Hi Ryan, 
if I understand this right this are the human motifs or does this also include mouse and is filtered by species in addMotifAnnotations?
I read in some other comment that Jeff said that mouse was also included, but on the viestra page it says V2.1 Beta human. 

Thank you

Tilo
